x-pack/libbeat/common/cloudfoundry/events.go

```diff
@@ -0,0 +1,526 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cloudfoundry
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/elastic/beats/libbeat/common"
+
+	"github.com/cloudfoundry/sonde-go/events"
+)
+
+// EventType defines the different event types that can be raised from RPLClient.
+type EventType uint
+
+// EventTypes from loggregator documented here: https://github.com/cloudfoundry/loggregator-api
```

Comment:
thanks for adding this, really helpful! It would look like we are using V1 types?

Reply:
Yes the code always converts from V2 types to V1. V2 doesn't have any more events then V1 just a different structure.